install

```diff
@@ -17,6 +17,11 @@ if [ $EUID != 0 ]; then
   exit
 fi
 
+if [ ! -d "/usr/local/bin" ] || [ -L "/usr/local/bin" ]; then
+  echo "WARNING: /usr/local/bin does not exist, creating it for you."
+  mkdir -m 755 /usr/local/bin
+fi
```

Comment:
Any chance the `mkdir` might fail if `/usr/local` also doesn't exist? On Linux, I'd add the `-p` flag - not sure it works the same on macOS?

Reply:
yes, the flag is the same in BSD mkdir